native/src/widget/text.rs

```diff
@@ -31,6 +31,21 @@ pub struct Text<Renderer: self::Renderer> {
     vertical_alignment: VerticalAlignment,
 }
 
+impl<Renderer: self::Renderer> Clone for Text<Renderer> {
+    fn clone(&self) -> Self {
+        Self {
+            content: self.content.clone(),
+            size: self.size,
+            color: self.color,
+            font: self.font,
+            width: self.width,
+            height: self.height,
+            horizontal_alignment: self.horizontal_alignment,
+            vertical_alignment: self.vertical_alignment,
+        }
+    }
+}
```

Comment:
Maybe we could move this implementation to the end of the file, as it's mostly boilerplate.

Reply:
I considered that but I wasn't sure if that would be preferred at the time. Will do!